.github/file-filters.yml

```diff
@@ -0,0 +1,4 @@
+# This is used by the action https://github.com/dorny/paths-filter
+
+high_risk_code: &high_risk_code
+  - '.github/file-filters.yml'
```

Comment:
- this is for testing and will be removed before merging
- need to align on set of potentially higher-risk files before merging as well
- use full repo paths, can check [getsentry/sentry-cocoa@main/.github/file-filters.yml](https://github.com/getsentry/sentry-cocoa/blob/main/.github/file-filters.yml?rgh-link-date=2024-09-30T06%3A52%3A52Z) for inspiration

Reply:
I think a good candidates are

- https://github.com/getsentry/sentry-react-native/blob/933804ec2ad2153b9bd42d91bff2bb69d7f2a672/packages/core/src/js/NativeRNSentry.ts
  - Represents the native module interface, when changes usually we see new issue about failed build due to outdated pods for example 
- https://github.com/getsentry/sentry-react-native/blob/ddcabffc8bd015392f28deb4f685ed358889c77e/packages/core/scripts/sentry-xcode.sh and https://github.com/getsentry/sentry-react-native/blob/ddcabffc8bd015392f28deb4f685ed358889c77e/packages/core/sentry.gradle
  - automatic upload scripts currently covered only by e2e build tests
  - missing unit tests at the moment